assets/js/script.js

```diff
@@ -0,0 +1,23 @@
+$(document).ready(function () {
+    $("#access-button").click(function () {
```

Comment:
U JS-u cemo koristiti jednostruke navodnike
```suggestion
    $('#access-button').click(function () {
```

Reply:
Ovo treba primeniti na celi fajl